### PR TITLE
Add Conversation API. Add info on generating JWTs.

### PR DESCRIPTION
**Pull request by tbedford**
## Description

* Fix issue #1143
* Adds additional link from webhooks topic to Application topic.

### CODE REVIEW

File: _documentation/concepts/guides/authentication.md [hunk 1]
```diff
@@ -120,6 +121,30 @@ Name | Description | Required
 
 If you are not using a Nexmo library you should refer to [RFC 7519](https://tools.ietf.org/html/rfc7519) to implement JWT.
 
+### Using the Nexmo CLI to generate JWTs
+
+The Nexmo CLI is one way of creating a JWT. The general syntax is:
+
+``` shell
```
Comment (mheap) [inline _documentation/concepts/guides/authentication.md]: I don't believe there should be a space here

Comment (tbedford) [reply _documentation/concepts/guides/authentication.md]: Emacs GFM mode puts a space in automatically so I assume that works. Is the codeblock failing to render correctly in some way?

File: _documentation/concepts/guides/authentication.md [hunk 2]
```diff
@@ -120,6 +121,30 @@ Name | Description | Required
 
 If you are not using a Nexmo library you should refer to [RFC 7519](https://tools.ietf.org/html/rfc7519) to implement JWT.
 
+### Using the Nexmo CLI to generate JWTs
+
+The Nexmo CLI is one way of creating a JWT. The general syntax is:
```
Comment (marklewin) [inline _documentation/concepts/guides/authentication.md]: I'm cringing at my pedantry here, but I think we should try and be as clear as possible at all times, particularly now that we're writing for translation.

The Nexmo CLI isn't one way of creating a JWT, it's a suite of commands for doing all sorts of Nexmo goodness. One way to create a JWT is to use the Nexmo CLI. Or the Nexmo CLI provides a command for creating a JWT.

Hope that makes sense!

File: _documentation/concepts/guides/authentication.md (hunk 1 above)
Comment (mheap) [reply _documentation/concepts/guides/authentication.md]: You're right! It does support a space in the commonmark spec https://spec.commonmark.org/0.12/#fenced-code-blocks

**Pull request merged**